qa-test-app/src/main/res/layout/layout_drawer_menu_nav_view_custom.xml

```diff
@@ -69,13 +69,31 @@
                 android:text="Use Custom Map"
                 android:textAllCaps="true" />
         </TableRow>
+        <TableRow android:padding="5dp">
+            <androidx.appcompat.widget.SwitchCompat
+                android:id="@+id/toggleCustomInfoPanel"
+                android:layout_weight="1"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="Custom Info Panel Layout"
+                android:textAllCaps="true" />
+        </TableRow>
+        <TableRow android:padding="5dp">
```

Comment:
I also think we should remove this toggle switch completely. It's a bit annoying that I see the `InfoPanel` and it's asking me to `pull up for details`; but when I try to it doesn't work because the functionality is disabled.

Reply:
What if I just change the message to "Custom header view for Free Drive mode".

Reply:
I mean you could do that, but the message is not the only thing that made me try to pull it. I wanted to, to see what the contents have. TBH, I am not seeing the value add of this switch just yet unless you already have something in mind that I am missing.

Reply:
This switch allows us testing of a custom InfoPanel content binding in any NavigationState. Regardless of toggleBottomSheetFD value.

Anyways this is our development/sandbox app. Please ignore all code being written here. It's only for testing.